engine/src/main/java/org/terasology/engine/core/ComponentSystemManager.java

```diff
@@ -68,7 +68,7 @@ public void loadSystems(ModuleEnvironment environment, NetworkMode netMode) {
         ListMultimap<Name, Class<?>> systemsByModule = ArrayListMultimap.create();
         for (Class<?> type : environment.getTypesAnnotatedWith(RegisterSystem.class)) {
             if (!ComponentSystem.class.isAssignableFrom(type)) {
-                logger.error("Cannot load {}, must be a subclass of ComponentSystem", type.getSimpleName());
+                logger.atError().addArgument(type.getSimpleName()).log("Cannot load {}, must be a subclass of ComponentSystem");
```

Comment:
Only called in error case and likelihood that call "in vain" is small due to error log-level
```suggestion
                logger.error("Cannot load {}, must be a subclass of ComponentSystem", type.getSimpleName()); //NOPMD
```